### PR TITLE
Integrated API service for check-in form

### PR DESCRIPTION
**Pull request by caroljli**
* Implemented API service for check-in form through `api.service.ts` (POST)
* Values are stored in check-in modal component
* Sends as param map -- cannot send type double as param so must convert manually in backend

### CODE REVIEW

File: shopsafe-frontend/src/environments/environment.ts [hunk 1]
```diff
@@ -3,7 +3,10 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+
+  // URL of development API
+  apiUrl: 'http://localhost:8080'
```
Comment (glstewart17) [inline shopsafe-frontend/src/environments/environment.ts:9]: Just wondering, have you tried it yet, or is that next?

**Review by glstewart17 — APPROVED** [review]
Looks good to me.

File: shopsafe-frontend/src/app/api/api.service.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  /**
+   * Creates new check-in for specific store ID
+   * @param storeId ID of the store that the check-in occurs for
+   * @returns Observable 
+   */
+  public createCheckIn(storeId: string, busy: number, line: number, hygiene: number, mask: number): Observable<Object> {
+    let params = new HttpParams();
+
+    // FIXME: Must convert to double in the backend
```
Comment (Raulp8) [inline shopsafe-frontend/src/app/api/api.service.ts:32]: done !

File: shopsafe-frontend/src/environments/environment.ts (hunk 1 above)
Comment (Raulp8) [inline shopsafe-frontend/src/environments/environment.ts:9]: Yeah, does this work when we deploy?

**Review by Raulp8 — APPROVED** [review]
looks good to me

File: shopsafe-frontend/src/environments/environment.ts (hunk 1 above)
Comment (caroljli) [reply shopsafe-frontend/src/environments/environment.ts:9]: There's a different environment file for production versus development, so we will prob have to map the URL differently i in `environment.prod.ts` (opposed to `environment.ts`). This is a good point though, I'll look into it.

**Pull request merged**